src/components/Rolls/index.tsx

```diff
@@ -399,32 +370,16 @@ const SideBar: React.FC = (): JSX.Element => {
   )
 }
 
-const BountyCard: React.FC<{
-  title
-  description
-  tags
-  project
-  createdAt
-  startDate
-  estimatedTime
-  skill
-  skillLevel
-  reward
-  numOfApplicants
-  isFavorite
-}> = ({
+const BountyCard: React.FC<Partial<Bounty>> = ({
   title,
-  description,
+  summary,
   tags,
   project,
   createdAt,
-  startDate,
```

Comment:
as said in another comment, we have an estimated start date and estimated time it should take to complete in the bounty cards UI on figma. We may also have filters for them as well. That way people can try to plan for bounties starting at a particular time and understand capacity.